examples/tensorflow-profiling-examples/example_test.py

```diff
@@ -0,0 +1,112 @@
+import unittest
+from unittest.mock import MagicMock
+from unittest.mock import patch
+
+import numpy as np
+import pandas as pd
+import tensorflow as tf
+from tensorflow.python.keras import testing_utils
+from tensorflow.python.keras import initializers
+
+import example
+
+
+class LinearBlockFullTest(tf.test.TestCase):
+    def test_basic(self):
+       testing_utils.layer_test(example.LinearBlockFull, input_shape=(4, 32))
+
+    def test_output(self):
+        batch_size, dim, output_dim = (3, 4, 2)
+        testing_utils.layer_test(
+            example.LinearBlockFull,
+            kwargs={'units': output_dim},
+            input_data=np.ones((batch_size, dim)),
+            expected_output_dtype='float32'
+        )
+
+
+class LinearBlockTest(tf.test.TestCase):
+    def test_shape_default(self):
+        x = np.ones((4, 32))
+        layer = example.LinearBlock()
+        output = layer(x)
+        self.assertAllEqual(output.shape, (4, 32))
+
+    def test_shape(self):
+        batch_size, dim = (4, 15)
+        x = np.ones((batch_size, dim))
+        layer = example.LinearBlock(dim)
+        output = layer(x)
+        self.assertAllEqual(output.shape, (batch_size, dim))
+
+    def test_shape_multidim(self):
+        batch_size, dim = (4, 15)
+        x = np.ones((batch_size, 7, dim))
+        layer = example.LinearBlock(dim)
+        output = layer(x)
+        self.assertAllEqual(output.shape, (batch_size, 7, dim))
+
+    def test_output(self):
+        dim = 4
+        batch_size = 3
```

Comment:
nit: inconsistent with the other tests where you initialize batch_size and dim on the same line

Reply:
fixed